zmq/README.md

```diff
@@ -0,0 +1,32 @@
+# ZeroMQ
```

Comment:
Model the `README` after the [README for Python](https://github.com/unikraft/static-pie-apps/pull/10) or [nginx](https://github.com/unikraft/static-pie-apps/tree/master/nginx).

Add package requirements.

Reply:
G++ is required, `libbsd-dev` is required.